axelrod/tests/unit/test_interaction_utils.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+import unittest
+import axelrod.interaction_utils as iu
+from axelrod import Actions
+
+C, D = Actions.C, Actions.D
+
+
+class TestMatch(unittest.TestCase):
+    interactions = [
```

Comment:
The spacing in this section seems like a bit much -- surely some of these fit in 80 characters. OTOH I'm not sure what PEP8 says in this cases....